.circleci/base_config.yml

```diff
@@ -9,7 +9,7 @@ parameters:
     default: "devel"
   build-targets:
     type: string
-    default: "arangod arangosh arangoimport arangoexport arangodump arangorestore arangobench"
+    default: "arangod arangosh arangoimport arangoexport arangodump arangorestore arangobench frontend"
```

Comment:
@pluma4345 we don't need this frontend here because CircleCI tests don't use UI.

Reply:
On the other hand it's interesting that still Swagger is eliminated as well and this frontend target is technically dependent.

Reply:
@pluma4345 to add these tools to the build environment they need to be added to the Dockerfile `scripts/docker/build/arangodb-build.Dockerfile` and the docker image has to be re-built (which at this point is still a manual process), and the docker image has to be added with the correct tag to the circleci config.

it looks like nodejs is already installed currently though (because the linting step uses eslint)

I can make these changes tomorrow if noone beats me to it.